### PR TITLE
Update version requirements and wording

### LINKED ISSUE #451: Update minimum required IntelliJ version in docs
We now require IntelliJ IDEA 2023.1 as a minimum. We support any fix releases for 2023.1 (2023.1.x). We should update the README and user guide accordingly.

### PR DESCRIPTION
**Pull request by evie-lau**
Resolves #451

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +64,8 @@ Editing assistance for configuration and application files is provided through t
 
 Projects with the Liberty Maven Plugin or Liberty Gradle Plugin configured will appear in the Liberty tool window on the sidebar. If the tool window is not enabled by default, you can view it by selecting **View > Tool Windows > Liberty**.
 
+> Note: Make sure the **JAVA_HOME** system environment variable is set to define the JDK for running Liberty dev mode.
```
Comment (evie-lau) [inline README.md:67]: @dmuelle Please review this latest change reflecting our Slack discussion.

Removed the Settings section in this README as it's covered in user-guide, but mentioned this note here as it is important.

Comment (dmuelle) [reply README.md:68]: do we need to mention Java 17 or later here? Also the previous statement said

> The **JAVA_HOME** system environment variable in the current terminal is used when you run Liberty dev mode in any project

but the user guide doesn't mention Java home

File: README.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
 # Liberty Tools for IntelliJ IDEA
 
-> Note: Starting with the [0.0.8 early release](https://github.com/OpenLiberty/liberty-tools-intellij/releases/tag/0.0.8), Java 17 (bundled with IntelliJ IDEA version 2022.2+) and a minimum version of IntelliJ IDEA version 2022.2 are required to run Liberty Tools for IntelliJ IDEA.
+<!-- Make sure to also update the #Requirements section below, and in user-guide.md -->
+> Note: Requires **IntelliJ IDEA 2023.1** or later, and **Java 17** or later. 
+>
+> This page provides an overview of Liberty Tools for IntelliJ.
+> For more details, check the [user-guide](docs/user-guide.md).
```
Comment (dmuelle) [inline README.md:7]:
```suggestion
> For minimum requirements information and detailed instructions on how to use Liberty Tools, check the [user-guide](docs/user-guide.md).
```

File: README.md (hunk 1 above)
Comment (evie-lau) [reply README.md:68]: Here is fine for the user to use a <17 version of Java, as it is used for running Liberty dev mode.

The plugin runs with the IntelliJ-bundled Java.

**Review by TrevCraw — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,15 +1,17 @@
 # Liberty Tools for IntelliJ IDEA
 
-> Note: Starting with the [0.0.8 early release](https://github.com/OpenLiberty/liberty-tools-intellij/releases/tag/0.0.8), Java 17 (bundled with IntelliJ IDEA version 2022.2+) and a minimum version of IntelliJ IDEA version 2022.2 are required to run Liberty Tools for IntelliJ IDEA.
+<!-- Make sure to also update the #Requirements section below, and in user-guide.md -->
+> Note: Requires **IntelliJ IDEA 2023.1** or later, and **Java 17** or later. 
+>
+> This page provides an overview of Liberty Tools for IntelliJ.
```
Comment (TrevCraw) [inline README.md:6]:
```suggestion
> This page provides an overview of Liberty Tools for IntelliJ IDEA.
```

**Pull request merged**